datadog_checks_base/tox.ini

```diff
@@ -6,6 +6,12 @@ envlist =
 
 [testenv]
 dd_check_style = true
+dd_check_types = true
+dd_mypy_args =
+    --py2
```

Comment:
py2?

Reply:
This flag should be used for code that needs to be Python 2.7 compatible. See https://mypy.readthedocs.io/en/stable/python2.html

I'm thinking we may want to add it to our base `mypy.ini` config file… Can treat that separately.